index.js

```diff
@@ -0,0 +1,11 @@
+import Clipboard from './src/clipboard';
+import ClipboardAction from './src/clipboard-action';
+import DelegateEvents from 'delegate-events';
+import TinyEmitter from 'tiny-emitter';
+
+// expose bundled libraries
+Clipboard.ClipboardAction = ClipboardAction;
+Clipboard.DelegateEvents = DelegateEvents;
+Clipboard.TinyEmitter = TinyEmitter;
```

Comment:
I personally wouldn't expose internal deps that much on a entrypoint. Local scope brings more maintainability.

Which  means:
```js
import Clipboard from './src/clipboard';
```
only and internal deps whenever it's required on a specific file.